homework sprint 10

leskiyy opened the pull request:
Реализовано ТЗ проекта №10

leskiyy committed:
homework sprint 10

leskiyy committed:
homework sprint 10
fix checkstyle error in tests

Sla-als commented on src/main/java/ru/yandex/practicum/filmorate/model/User.java:
```diff
@@ -0,0 +1,31 @@
+    private String name;
+
+    @NonNull
+    @Past
```
В модели User для даты рождения (birthday) используется аннотация @Past. По требованиям ТЗ дата рождения «не может быть в будущем», то есть сегодняшняя дата должна быть допустима. 

Аннотация @Past не пропускает текущую дату (она требует строго прошедшую), а в ТЗ достаточно запретить будущее.

Sla-als reviewed (commented):
Один небольшой комментарий, в остальном, отличная работа!

leskiyy committed:
homework sprint 10
fix: сменить валидацию поля LocalDate birthday класса User c @Past на @PastOrPresent
